relay-event-schema/src/protocol/contexts/trace.rs

```diff
@@ -161,6 +161,21 @@ pub struct Data {
     #[metastructure(field = "previousRoute", pii = "maybe", skip_serialization = "empty")]
     pub previous_route: Annotated<Route>,
 
+    #[metastructure(field = "messaging.destination.name")]
+    pub messaging_destination_name: Annotated<String>,
+
+    #[metastructure(field = "messaging.message.id")]
+    pub messaging_message_id: Annotated<String>,
+
+    #[metastructure(field = "messaging.message.receive.latency")]
+    pub messaging_message_receive_latency: Annotated<Value>,
+
+    #[metastructure(field = "messaging.message.retry.count")]
+    pub messaging_message_retry_count: Annotated<Value>,
+
+    #[metastructure(field = "messaging.message.body.size")]
+    pub messaging_message_body_size: Annotated<Value>,
```

Comment:
Have you discussed this schema with SDK teams? It seems like a separate `messaging` context could make more sense.

Reply:
We discussed some other options, but haven't thought about a new `messaging` context yet.

@cleptric What do you think?

Reply:
@jjbayer we use `span.data` for this module. As a transaction has no data attribute, it ends up on the trace context. I would like to avoid adding a one of context to the SDK if possible but also get the point about it in Relay 😬

Reply:
OK, if `contexts.trace.data` is the transaction field corresponding to `span.data` then this makes sense.